.github/actions/accessibility-test-jupyterlab/action.yml

```diff
@@ -76,26 +96,21 @@ runs:
         jupyter lab --version
       shell: bash -l {0}
 
-    - name: Install node dependencies 🧶
-      run: |
-        conda run --prefix $CONDA_PREFIX npm install
-        conda run --prefix $CONDA_PREFIX npx playwright install --with-deps chromium
-      shell: bash -l {0}
-      working-directory: testing/jupyterlab
-
     - name: Run tests  ✅
+      # we want the workflow to continue - i.e. we expect a11y errors so we want to diagnose and
+      # get the output summary
       continue-on-error: true
       run: |
         if [[ "${{ inputs.test_project }}" ]]
         then
-          conda run --prefix $CONDA_PREFIX npm test -- --project="${{ inputs.test_project }}"
+          ${{ env.npm_path }} test -- --project="${{ inputs.test_project }}"
```

Comment:
Why is this `${{ env.npm_path }}` needed? Should it be explained in a comment?

Reply:
Similar to using `conda run $CONDA_PREFIX`, we ensure we are using the correct package by calling the "system" `npm` executable directly (best to avoid ambiguous paths and confusions)

Reply:
can add a comment if you think is needed